### PR TITLE
Remove Language Model and API

### PR DESCRIPTION
**Pull request by shivamMg**
https://github.com/fossasia/open-event-orga-server/issues/818

@aviaryan Can you please take a look.

### CODE REVIEW

File: migrations/versions/26a83ff1e6ad_.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.drop_table('language')
+    op.drop_constraint(u'session_language_id_fkey', 'session', type_='foreignkey')
```
Comment (aviaryan) [inline migrations/versions/26a83ff1e6ad_.py]: op.drop_constraint .... move this line above op.drop_table to avoid db upgrade issues.

**Comment by aviaryan** [comment]
@shivamMg Can you add the new `language` string field in this PR. It should use the fields.Choice field, the same thing has been done with event type and topic.

**Comment by shivamMg** [comment]
@aviaryan Done!

File: docs/API.md [hunk 2]
```diff
@@ -86,7 +76,6 @@ Datatype, requirement and access-level has been defined for every model. Nested
 |**start_time** | string | Required | Public |
 |**end_time** | string | Required | Public |
 |**speakers** | Array[**SessionSpeaker**] | Optional | Public |
-|**language** | **SessionLanguage** | Optional | Public |
```
Comment (aviaryan) [inline docs/API.md]: Add back language field in docs :stuck_out_tongue_winking_eye:

**Comment by aviaryan** [comment]
Merging ..

**Pull request merged**